src/routes/DarkMode/+page.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	import DarkMode from '$lib/DarkMode.svelte';
+	import ExamplePage from '../ExamplePage.svelte';
+
+	const usage = '<DarkMode />';
```

Comment:
Even though its a pretty basic usage, I still think its worth having the extra component and importing usage using `?raw`
e.g.
```
import ExampleDarkMode from "./ExampleDarkMode.svelte";
import usage from "./ExampleDarkMode.svelte?raw";
```